### ISSUE TITLE
lxde/lxqt-config:  files without license

### ISSUE THREAD

**Issue by agaida** [lxde/lxqt#785]
lxqt-config-file-associations/mimetypeitemmodel.h: *No copyright* UNKNOWN @surlykke 
lxqt-config-file-associations/mimetypeitemmodel.cpp: *No copyright* UNKNOWN @surlykke 
lxqt-config-input/lxqt-config-input.h: *No copyright* UNKNOWN @PCMan 
lxqt-config-monitor/monitorpicture.cpp: *No copyright* UNKNOWN @selairi 

please add valid license headers, this is urgent, because without a license no debian package :P

**Comment by jleclanche** [comment]
Every file without a license header falls under the global license of the project. License headers are *not* required, by the way, so there is no reason this should block debian packaging.

Regardless, this is the last header we agreed on:

```c
/*
* Copyright (c) LXQt contributors.
*
* This file is part of the LXQt project. <http://lxqt.org>
* It is distributed under the LGPL 2.1 or later license.
* Please refer to the LICENSE file for a copy of the license, and
* the AUTHORS file for copyright and authorship information.
*/
```

**Comment by agaida** [comment]
@jleclanche: that depends - its a lot easier if the licenses of every file are declared in the file - otherwise one have to argue with ftp-masters - and one will loose with the chance of > 95%

**Comment by agaida** [comment]
fixed with times ago with lxde/lxqt-config #811656
